### DISCUSSION TITLE
The first Closeread Prize is here!

### DISCUSSION THREAD

**Discussion by jimjam-slam** [qmd-lab/closeread#125] [Announcements]
Calling all scrollytellers, storytellers and budding data journalists! The first Closeread Prize is here!

The kind folks at [Posit](https://posit.co) have teamed up with Andrew and James to give away some awesome prizes to the most creative, technically accomplished and engaging scrollytelling examples using Closeread and Quarto.

**See the announcement blog post, with rules, submission criteria, entry form and inspiration here:**

**https://posit.co/blog/closeread-prize-announcement/**

We're super excited to see what you all make, but we also want to use this as a chance to see what you love about Closeread and what's causing you grief.

If you get stuck and need help fro the community, think something could be improved or just want to show off your prize entry, don't hesitate to [file an issue](https://github.com/qmd-lab/closeread/issues/new) or [start a discussion](https://github.com/qmd-lab/closeread/discussions/new)!

You have until **[midnight, ~~December 15~~ January 5 Anywhere on Earth](https://dateful.com/convert/aoe-anywhere-on-earth?t=12am&d=2025-01-05)** to get those entries in. Find the entry form on Posit's website from the blog post above!

If you've already been using Closeread, you should also [check out our new releases](https://github.com/qmd-lab/closeread/releases): we have a heap of new features and bug fixes already, and more will be on the way as the prize progresses! You can always update the version of Closeread you're using by installing it again.

Good luck, and have fun!

**Comment by fsemecurbe** [comment]
Hi @jimjam-slam,

do you know when we will have the results of the Prize? 

Thank you very much.

**Comment by andrewpbray** [comment]
Hi @fsemecurbe , Very soon! We have all of the scores in from the panel of judges and preparing the awards announcement. We're aiming for next Monday.

Thanks for your patience!

**Reply by fsemecurbe** [reply]
I m sorry, I don' want to bother, I am looking forward to knowing the results because I try to convince my boss that is the good tool to replace https://vizagreste.agriculture.gouv.fr/evolution-du-nombre-d-exploitations.html

Thank in advance
